pkg/interactive/validation.go

```diff
@@ -63,14 +64,21 @@ func IsURL(val interface{}) error {
 }
 
 func IsValidHostname(val interface{}) error {
-	if !_isValidHostname(val.(string)) {
+	hostname := val.(string)
+	if hostname == "github.com" || strings.HasSuffix(hostname, ".github.com") {
+		return fmt.Errorf(fmt.Sprintf("'%s' hostname cannot be equal [*.]github.com", hostname))
```

Comment:
@mnecas Missing a `to` in the error message e.g `cannot be equal to [*.]github.com`

Reply:
+1